lib/copilot-components/addon/components/foo-bar/style.scss

```diff
@@ -1,4 +1,4 @@
 section {
   background-color: pink;
- // color: $button-text-dark;
+  color: $button-text-dark;
```

Comment:
Yay variable from `platform-styleguide` can be pulled into in-repo component pod style!